src/journal_rss/app.py

```diff
@@ -40,7 +40,7 @@ def on_startup():
 
 @app.get('/', response_class=HTMLResponse)
 async def index(request: Request):
-    return templates.TemplateResponse('index.html', {"request": request})
+    return templates.TemplateResponse('shared/index.html', {"request": request})
```

Comment:
i like the restructuring! super minor thing but help me understand what you're thinking here - i was using 'shared' as root template(s) that individual pages extend from, but since this template is now just the template for the index page (ie. not used again for the eg. journal feed pages) then it's not exactly shared? if we want this to be in a directory maybe we call it 'pages' or something?

Reply:
Ah, fair enough. Totally agree, having a folder called pages would be good then!